src/cr/cube/crunch_cube.py

```diff
@@ -837,3 +838,35 @@ def index(self, axis, weighted=True):
             margin = margin[:, np.newaxis]
 
         return np.true_divide(props, margin)
+
+    @property
+    def z_scores(self):
+
+        if self.has_mr:
+            return self._mr_z_scores
+
+        counts = self.as_array()
+        total = self.margin()
+        p_col = self.margin(axis=0) / total
+        p_row = self.margin(axis=1) / total
+        expected_counts = chi2_contingency(counts)[3]
+        residuals = counts - expected_counts
+        V = expected_counts * ((1 - p_row[:, np.newaxis]) * (1 - p_col))
```

Comment:
Is there a good reason not to do this in count space vs probability space? Only to do less floating point division and less possibility for rounding error. 
```
colsum = self.margin(axis=0)
rowsum = self.margin(axis=1)
V = np.outer(colsum, rowsum) * np.outer(total-colsum, total-rowsum) / n**3
```
(as usual, I might be transposed)

Reply:
For this, I just followed what's implemented in R. I'll try your way, and see what happens :+1: